src/components/AnonymousChat/BaseChannelItem.tsx

```diff
@@ -39,20 +38,9 @@ const BaseChannelItem: (props: BaseChannelItemProps) => React.ReactElement = ({
   onPress = () => {
     console.log('onPress');
   },
+  showFollowingButton = false,
   dbAnonUserInfo = null
 }) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the context and the related logic for checking if a user is being followed simplifies the component. However, it's important to ensure that the `showFollowingButton` prop is being correctly set in the parent component. This change assumes that the parent component has access to the necessary context or state to determine whether the follow button should be shown.
```diff
-  const [profileContext] = (React.useContext(Context) as unknown as any).profile;
-  const [followContext] = (React.useContext(Context) as unknown as any).following;
-  const {myProfile} = profileContext;
-  let isFollowing = false;
-  if (type === BaseChannelItemTypeProps.SIGNED_PM) {
-    const members = postMaker?.members || postMaker?.channel?.members;
-    const targetUser = members?.find((member) => member?.user_id !== myProfile?.user_id)?.user;
-    isFollowing = Boolean(
-      followContext?.users?.find((user) => user?.user_id_followed === targetUser?.id)
-    );
-  }
+  showFollowingButton = false,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->